favorite_files.py

```diff
@@ -8,7 +8,7 @@
 import sublime
 import sublime_plugin
 from os.path import join, exists
-from FavoriteFiles.favorites import Favorites
+from FavoriteFiles.favorites import Favorites, FavFileMgr
```

Comment:
FavFileMgr was not originally intended to be exposed. It was supposed to be a helper for Favorites to perform actions.  Is it necessary to expose it here?  Can't we just add access functions to Favorites that relies on FavFileMgr?

Reply:
I don't know if it's necessary, but I did it because it kept saving the key in the project json instead of the global one, when switching from project to global. If you don't want the import, I can just make file operations there, or put those read_file and write_file outside of FavFileMgr, it's not needed that hey stay inside there.

Reply:
I don't understand, what key is not going to the project specific settings?  And if this is related to remembering project specific settings, this shouldn't be in this pull as that is a separate feature that should be implemented in a separate pull.  There's is enough complexity going on in this pull as is.

Reply:
I'll just remove this stuff for now